CRITICAL - 2022-09-16 10:05:25 --> Invalid file: blog/v_blog_detail.php
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('blog/v_blog_detail.php')
 2 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('blog/v_blog_detail', [], true)
 3 APPPATH\Views\layout\v_content.php(3): view('blog/v_blog_detail')
 4 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\layout\\v_content.php')
 5 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 6 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('layout/v_content.php', [], true)
 7 APPPATH\Views\layout\v_wrapper.php(5): view('layout/v_content.php')
 8 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\layout\\v_wrapper.php')
 9 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
10 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('layout/v_wrapper', [], true)
11 APPPATH\Controllers\Admin\Blog.php(33): view('layout/v_wrapper', [...])
12 SYSTEMPATH\CodeIgniter.php(904): App\Controllers\Admin\Blog->detail('edit', '1')
13 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Blog))
14 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
15 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
16 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
CRITICAL - 2022-09-16 10:14:35 --> Undefined variable $value
in APPPATH\Views\blog\v_blog_edit.php on line 40.
 1 APPPATH\Views\blog\v_blog_edit.php(40): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $value', 'APPPATH\\Views\\blog\\v_blog_edit.php', 40)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\blog\\v_blog_edit.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('blog/v_blog_edit', [], true)
 5 APPPATH\Views\layout\v_content.php(3): view('blog/v_blog_edit')
 6 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\layout\\v_content.php')
 7 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 8 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('layout/v_content.php', [], true)
 9 APPPATH\Views\layout\v_wrapper.php(5): view('layout/v_content.php')
10 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\layout\\v_wrapper.php')
11 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
12 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('layout/v_wrapper', [], true)
13 APPPATH\Controllers\Admin\Blog.php(65): view('layout/v_wrapper', [...])
14 SYSTEMPATH\CodeIgniter.php(904): App\Controllers\Admin\Blog->edit('1')
15 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Blog))
16 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
17 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
18 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
ERROR - 2022-09-16 21:56:49 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\ci4\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\ci4\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'root', '', 'interiorpadang-...', 3306, '', 0)
#1 C:\xampp\htdocs\ci4\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\ci4\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\ci4\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ci4\system\Model.php(222): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ci4\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\ci4\app\Models\BeritaModel.php(13): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\ci4\app\Controllers\Admin\Berita.php(19): App\Models\BeritaModel->getBlog()
#8 C:\xampp\htdocs\ci4\system\CodeIgniter.php(904): App\Controllers\Admin\Berita->index()
#9 C:\xampp\htdocs\ci4\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Berita))
#10 C:\xampp\htdocs\ci4\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\ci4\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\ci4\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\ci4\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\ci4\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\ci4\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\ci4\system\Model.php(222): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\ci4\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\ci4\app\Models\BeritaModel.php(13): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\ci4\app\Controllers\Admin\Berita.php(19): App\Models\BeritaModel->getBlog()
#7 C:\xampp\htdocs\ci4\system\CodeIgniter.php(904): App\Controllers\Admin\Berita->index()
#8 C:\xampp\htdocs\ci4\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Berita))
#9 C:\xampp\htdocs\ci4\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ci4\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-09-16 21:56:49 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `blog`', [], false)
 3 SYSTEMPATH\Model.php(222): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Models\BeritaModel.php(13): CodeIgniter\BaseModel->findAll()
 6 APPPATH\Controllers\Admin\Berita.php(19): App\Models\BeritaModel->getBlog()
 7 SYSTEMPATH\CodeIgniter.php(904): App\Controllers\Admin\Berita->index()
 8 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Berita))
 9 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
ERROR - 2022-09-16 21:56:54 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\ci4\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\ci4\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'root', '', 'interiorpadang-...', 3306, '', 0)
#1 C:\xampp\htdocs\ci4\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\ci4\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\ci4\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ci4\system\Model.php(222): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ci4\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\ci4\app\Models\BlogModel.php(14): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\ci4\app\Controllers\Admin\Blog.php(19): App\Models\BlogModel->getBlog()
#8 C:\xampp\htdocs\ci4\system\CodeIgniter.php(904): App\Controllers\Admin\Blog->index()
#9 C:\xampp\htdocs\ci4\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Blog))
#10 C:\xampp\htdocs\ci4\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\ci4\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\ci4\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\ci4\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\ci4\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\ci4\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\ci4\system\Model.php(222): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\ci4\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\ci4\app\Models\BlogModel.php(14): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\ci4\app\Controllers\Admin\Blog.php(19): App\Models\BlogModel->getBlog()
#7 C:\xampp\htdocs\ci4\system\CodeIgniter.php(904): App\Controllers\Admin\Blog->index()
#8 C:\xampp\htdocs\ci4\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Blog))
#9 C:\xampp\htdocs\ci4\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ci4\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-09-16 21:56:54 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `blog`', [], false)
 3 SYSTEMPATH\Model.php(222): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Models\BlogModel.php(14): CodeIgniter\BaseModel->findAll()
 6 APPPATH\Controllers\Admin\Blog.php(19): App\Models\BlogModel->getBlog()
 7 SYSTEMPATH\CodeIgniter.php(904): App\Controllers\Admin\Blog->index()
 8 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Blog))
 9 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
ERROR - 2022-09-16 21:56:58 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\ci4\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\ci4\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'root', '', 'interiorpadang-...', 3306, '', 0)
#1 C:\xampp\htdocs\ci4\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\ci4\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\ci4\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ci4\system\Model.php(222): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ci4\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\ci4\app\Models\BlogModel.php(14): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\ci4\app\Controllers\Admin\Blog.php(19): App\Models\BlogModel->getBlog()
#8 C:\xampp\htdocs\ci4\system\CodeIgniter.php(904): App\Controllers\Admin\Blog->index()
#9 C:\xampp\htdocs\ci4\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Blog))
#10 C:\xampp\htdocs\ci4\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\ci4\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\ci4\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\ci4\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\ci4\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\ci4\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\ci4\system\Model.php(222): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\ci4\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\ci4\app\Models\BlogModel.php(14): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\ci4\app\Controllers\Admin\Blog.php(19): App\Models\BlogModel->getBlog()
#7 C:\xampp\htdocs\ci4\system\CodeIgniter.php(904): App\Controllers\Admin\Blog->index()
#8 C:\xampp\htdocs\ci4\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Blog))
#9 C:\xampp\htdocs\ci4\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ci4\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-09-16 21:56:58 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `blog`', [], false)
 3 SYSTEMPATH\Model.php(222): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Models\BlogModel.php(14): CodeIgniter\BaseModel->findAll()
 6 APPPATH\Controllers\Admin\Blog.php(19): App\Models\BlogModel->getBlog()
 7 SYSTEMPATH\CodeIgniter.php(904): App\Controllers\Admin\Blog->index()
 8 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Blog))
 9 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
